gravitee-apim-portal-webui-next/src/app/api/api-details/api-tab-subscriptions/subscriptions-details/subscriptions-details.component.scss

```diff
@@ -18,6 +18,7 @@
     display: flex;
     align-items: center;
     padding-left: 8px;
+    cursor: pointer;
```

Comment:
why not add <a> tags instead of <div> N

Reply:
The `<a>` tag changes the color of the arrow + text. So the simplest version would be just to keep the `<div>` and then add the `cursor: pointer`. 

With `<a>` tag:
![Screenshot 2024-07-15 at 18 08 11](https://github.com/user-attachments/assets/4df4a9fb-d992-4c8b-8c02-8d3b823139b8)